scripts/binary/build.ts

```diff
@@ -68,15 +68,23 @@ export async function buildCypressApp (options: BuildCypressAppOpts) {
   // Copy Packages: We want to copy the package.json, files, and output
```

Comment:
why re-name from `.patch` to `.dev.patch`?

Reply:
Do the patches in `packages/socket/patches` & `packages/serverp/patches` also need to be renamed to `.dev.patch`?

Reply:
https://github.com/cypress-io/cypress/pull/19239/files#diff-d4d26f3d914f934bada7003fc6d0be8ebc4a24777a765c0302a1ece4892c2cf2R85

It's the way you're supposed to name them for `devDependencies`

Reply:
Ahhhh okay 👍🏻

Reply:
Oops, had the wrong link. But here's the docs: https://github.com/ds300/patch-package#dev-only-patches

And no, the socket & server ones are prod dependencies, so they remain `.patch`